src/Compilers/Core/Portable/Compilation/IStatement.cs

```diff
@@ -101,19 +94,19 @@ public enum CaseKind
         /// <summary>
         /// Indicates case x in C# or Case x in VB.
         /// </summary>
-        SingleValue,
+        SingleValue = 0x0,
```

Comment:
FxCop suggests all the enums should have a None = 0x0 to indicate uninitialized. Should we enforce that?
https://msdn.microsoft.com/en-us/library/ms182149.aspx